pkg/application/service/services.go

```diff
@@ -11,8 +11,6 @@ import (
 type InformerService interface {
 	GetMasterUserRecord(name string) (*toolchainv1alpha1.MasterUserRecord, error)
 	GetSpace(name string) (*toolchainv1alpha1.Space, error)
-	GetToolchainStatus() (*toolchainv1alpha1.ToolchainStatus, error)
-	GetUserSignup(name string) (*toolchainv1alpha1.UserSignup, error)
 	ListSpaceBindings(reqs ...labels.Requirement) ([]toolchainv1alpha1.SpaceBinding, error)
```

Comment:
just for my understanding - why dropping only these two methods and not all the ones from the resource_provider.go ?

Reply:
This is the InformerService interface, not ResourceProvider. 
I dropped only those methods that are not used anywhere now, the rest is still used in the proxy logic

Reply:
I see, maybe I got lost/confused by the fact that resource provider and informerservice had overlapping methods 😕  :
```
type ResourceProvider interface {
	GetMasterUserRecord(name string) (*toolchainv1alpha1.MasterUserRecord, error)
	GetToolchainStatus() (*toolchainv1alpha1.ToolchainStatus, error)
	GetUserSignup(name string) (*toolchainv1alpha1.UserSignup, error)
	GetSpace(name string) (*toolchainv1alpha1.Space, error)
	ListSpaceBindings(reqs ...labels.Requirement) ([]toolchainv1alpha1.SpaceBinding, error)
}
```

Reply:
Yes, they had - InformerService was one of the implementations of ResourceProvider, but the functions were not used only for the purpose of the ResourceProvider, but also in other cases (in proxy). But don't worry, the time when I'll be able to drop the InfomerService is coming closer :-)

Reply:
Yeah thanks, and sorry for the confusion I got lost with all the abstractions 😄

Reply:
No worries, I get lost in all the abstraction every time when I'm working on it :smile: